File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/InlineObject13.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param groupId Group ID to query.
 */
data class InlineObject13 (

        @get:NotNull 
        @JsonProperty("groupId") val groupId: Long
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineObject13.scala

package org.openapitools.client.model


case class InlineObject13 (
    /* Group ID to query. */
    _groupId: Long
)
object InlineObject13 {
    def toStringBody(var_groupId: Object) =
        s"""
        | {
        | "groupId":$var_groupId
        | }
        """.stripMargin
}
